### PR TITLE
Fix command.adoc bugs

### PR DESCRIPTION
**Pull request by stoobie**
### Description of the Changes

- Moved and rewrote tips on setting network variable and gateways
- Fixed broken link

### Check List

- [ ] Changes have been done against dev branch, and PR does not conflict
- [ ] Docs have been updated
- [ ] PR title is follow the convention: `<docs/feat/fix/chore>(optional scope): <description>`, e.g: `fix: minor typos in code`

- - -
This change is [Reviewable](https://reviewable.io/reviews/starknet-community-libs/starknet-docs/64)

### CODE REVIEW

File: components/StarkNet/modules/develop/pages/CLI/commands.adoc [hunk 1]
```diff
@@ -1,20 +1,60 @@
...
+== Setting the StarkNet network environment
+
+You need to set your StarkNet network environment to use either testnet or Mainnet.
+
+You can set the environment using either a command-line option or an environment variable.
+
+Possible values are:
+
+`alpha-goerli`:: Sets the StarkNet network to testnet
+`alpha-mainnet`:: Sets the StarkNet network to testnet
```
Comment (bbrandtom) [inline components/StarkNet/modules/develop/pages/CLI/commands.adoc:19]: testnet -> mainnet

File: components/StarkNet/modules/develop/pages/CLI/commands.adoc [hunk 2]
```diff
@@ -1,20 +1,60 @@
...
+.Setting the environment using an environment variable
+
+Set the `STARKNET_NETWORK` environment variable as follows:
+
+[source,bash]
+----
+export STARKNET_NETWORK=<starknet_network>
+----
+
+For example, to use `testnet`, enter the following command:
```
Comment (bbrandtom) [inline components/StarkNet/modules/develop/pages/CLI/commands.adoc:39]: why testnet in code format?

File: components/StarkNet/modules/develop/pages/CLI/commands.adoc [hunk 3]
```diff
@@ -1,20 +1,60 @@
...
+When working with the CLI, you can manually set the read/write endpoints for the gateways that enable you to
+interact with StarkNet, by including the `--feeder_gateway_url` and `--gateway_url` options.
+
+The following are the endpoints for StarkNet testnet and Mainnet:
+
+* Testnet feeder gateway URL: https://alpha4.starknet.io/feeder_gateway/
+* Mainnet feeder gateway URL: https://alpha-mainnet.starknet.io/feeder_gateway/
+* Testnet gateway URL: https://alpha4.starknet.io/gateway/
+* Mainnet gateway URL: https://alpha-mainnet.starknet.io/gateway/
+
```
Comment (bbrandtom) [inline components/StarkNet/modules/develop/pages/CLI/commands.adoc:57]: Maybe add an example

File: ui/supplemental_ui/css/supplemental.css [hunk 4]
```diff
@@ -46,7 +46,7 @@ strong strong {
 .doc a code:hover {
   color: #104d92
 }
-
+z
```
Comment (bbrandtom) [inline ui/supplemental_ui/css/supplemental.css:49]: what is this?

**Review by stoobie — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/starknet-community-libs/starknet-docs/64)* status: 0 of 2 files reviewed, 4 unresolved discussions (waiting on @bbrandtom)

___
*[`components/StarkNet/modules/develop/pages/CLI/commands.adoc` line 19 at r1](https://reviewable.io/reviews/starknet-community-libs/starknet-docs/64#-NEFCN0h9Xlzm-ndXKBD-r1-19:-NEG4CXL6CPgeBOf81Ms:bt2vr1b) ([raw file](https://github.com/starknet-community-libs/starknet-docs/blob/ff85a1aed81a3e4a2547db09e8ad56041059b3ad/components/StarkNet/modules/develop/pages/CLI/commands.adoc#L19)):*
<details><summary><i>Previously, bbrandtom (Tom Brand) wrote…</i></summary><blockquote>

testnet -> mainnet
</blockquote></details>

Fixed.
___
*[`components/StarkNet/modules/develop/pages/CLI/commands.adoc` line 39 at r1](https://reviewable.io/reviews/starknet-community-libs/starknet-docs/64#-NEFCN0h9Xlzm-ndXKBD-r1-39:-NEG48f8CfGOwYdWACg9:b-vvuf93) ([raw file](https://github.com/starknet-community-libs/starknet-docs/blob/ff85a1aed81a3e4a2547db09e8ad56041059b3ad/components/StarkNet/modules/develop/pages/CLI/commands.adoc#L39)):*
<details><summary><i>Previously, bbrandtom (Tom Brand) wrote…</i></summary><blockquote>

why testnet in code format?
</blockquote></details>

Removed code format.
___
*[`components/StarkNet/modules/develop/pages/CLI/commands.adoc` line 57 at r1](https://reviewable.io/reviews/starknet-community-libs/starknet-docs/64#-NEFCN0h9Xlzm-ndXKBD-r1-57:-NEG5o3R2Db0jebNU_l4:b22b339) ([raw file](https://github.com/starknet-community-libs/starknet-docs/blob/ff85a1aed81a3e4a2547db09e8ad56041059b3ad/components/StarkNet/modules/develop/pages/CLI/commands.adoc#L57)):*
<details><summary><i>Previously, bbrandtom (Tom Brand) wrote…</i></summary><blockquote>

Maybe add an example
</blockquote></details>

Added an example, please check. Which commands include the `--feeder_gateway_url` and `-gateway_url`?
___
*[`ui/supplemental_ui/css/supplemental.css` line 49 at r1](https://reviewable.io/reviews/starknet-community-libs/starknet-docs/64#-NEFCN0h9Xlzm-ndXKBE-r1-49:-NEG4JhY79gKMxfrmD8Q:b-3lxi60) ([raw file](https://github.com/starknet-community-libs/starknet-docs/blob/ff85a1aed81a3e4a2547db09e8ad56041059b3ad/ui/supplemental_ui/css/supplemental.css#L49)):*
<details><summary><i>Previously, bbrandtom (Tom Brand) wrote…</i></summary><blockquote>

what is this?
</blockquote></details>

Ignore this. It fixes an error I saw in the style sheet.

**Review by bbrandtom — CHANGES_REQUESTED** [review]
> Reviewed 1 of 2 files at r1, 1 of 1 files at r3, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/starknet-community-libs/starknet-docs/64)* status: all files reviewed, 1 unresolved discussion (waiting on @stoobie)

___
*[`components/StarkNet/modules/develop/pages/CLI/commands.adoc` line 57 at r1](https://reviewable.io/reviews/starknet-community-libs/starknet-docs/64#-NEFCN0h9Xlzm-ndXKBD-r1-57:-NEG8P7x-engzM6L-zgC:bz7zepq) ([raw file](https://github.com/starknet-community-libs/starknet-docs/blob/ff85a1aed81a3e4a2547db09e8ad56041059b3ad/components/StarkNet/modules/develop/pages/CLI/commands.adoc#L57)):*
<details><summary><i>Previously, stoobie (Steve Goodman) wrote…</i></summary><blockquote>

Added an example, please check. Which commands include the `--feeder_gateway_url` and `-gateway_url`?
</blockquote></details>

`--feeder_gateway_url`
is for read commands such as `starknet get_code`

`--gateway_url`
is for write commands such as `starknet invoke`

**Review by stoobie — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/starknet-community-libs/starknet-docs/64)* status: 0 of 2 files reviewed, 1 unresolved discussion (waiting on @bbrandtom)

___
*[`components/StarkNet/modules/develop/pages/CLI/commands.adoc` line 57 at r1](https://reviewable.io/reviews/starknet-community-libs/starknet-docs/64#-NEFCN0h9Xlzm-ndXKBD-r1-57:-NEG9tuvAP9SNxH2-dB5:b-9hsb4v) ([raw file](https://github.com/starknet-community-libs/starknet-docs/blob/ff85a1aed81a3e4a2547db09e8ad56041059b3ad/components/StarkNet/modules/develop/pages/CLI/commands.adoc#L57)):*
<details><summary><i>Previously, bbrandtom (Tom Brand) wrote…</i></summary><blockquote>

`--feeder_gateway_url`
is for read commands such as `starknet get_code`

`--gateway_url`
is for write commands such as `starknet invoke`

</blockquote></details>

Please check now.

**Review by bbrandtom — CHANGES_REQUESTED** [review]
> Reviewed 1 of 2 files at r1, 1 of 1 files at r2, 2 of 2 files at r4, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/starknet-community-libs/starknet-docs/64)* status: all files reviewed, 1 unresolved discussion (waiting on @stoobie)

___
*[`components/StarkNet/modules/develop/pages/CLI/commands.adoc` line 89 at r4](https://reviewable.io/reviews/starknet-community-libs/starknet-docs/64#-NEGlld296gNt7vd8B46:-NEGlld296gNt7vd8B47:b8g4und) ([raw file](https://github.com/starknet-community-libs/starknet-docs/blob/5f523b11c6af759d5dc364ca8da69dfc9b8af6d5/components/StarkNet/modules/develop/pages/CLI/commands.adoc#L89)):*
> ```AsciiDoc
> ----
> $ starknet get_code --feeder_gateway_url https://alpha-mainnet.starknet.io/feeder_gateway/ <options>
> --gateway_url https://alpha-mainnet.starknet.io/gateway/
> ```

no need for this, as you've defined the feeder gateway

**Review by stoobie — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/starknet-community-libs/starknet-docs/64)* status: 1 of 2 files reviewed, 1 unresolved discussion (waiting on @bbrandtom)

___
*[`components/StarkNet/modules/develop/pages/CLI/commands.adoc` line 89 at r4](https://reviewable.io/reviews/starknet-community-libs/starknet-docs/64#-NEGlld296gNt7vd8B46:-NEHsT7CARbcOd-PPZQ3:b-896fix) ([raw file](https://github.com/starknet-community-libs/starknet-docs/blob/5f523b11c6af759d5dc364ca8da69dfc9b8af6d5/components/StarkNet/modules/develop/pages/CLI/commands.adoc#L89)):*
<details><summary><i>Previously, bbrandtom (Tom Brand) wrote…</i></summary><blockquote>

no need for this, as you've defined the feeder gateway
</blockquote></details>

Done.

**Pull request merged**